motor_control/Triple_HBridge/3PhaseMotorControl_UCC277.ino

```diff
@@ -0,0 +1,219 @@
+#include <ramping.h>
+
+int outHighPins[3] = { 10, 6, 4};
+int outLowPins[3] = {9, 5, 3};
+int inPins[3] = {0,1,2};
+int throttlePin = A0;
+bool pin1_state = 0;
+bool pin2_state = 0;
+bool pin3_state = 0;
+int motor_speed = 0;
+int speed_max = 0;
+int interrupt_time = 0;
+
+bool changing_state = false;
+bool interrupting = false;
+
+#define RAMP_SPEED 2;
+

```

Comment:
need ramp pointer declaration and rampingOption variable of type bool to use them in this code